public/translations/en-US.json

```diff
@@ -47,6 +47,7 @@
         "negative": "The number must be negative",
         "greater_than": "The number must be greater than {{base}}",
         "greater_than_or_equal": "The number must be greater than or equal to {{base}}",
+        "length_greater_than_or_equal": "Choose at least {{count}} items",
```

Comment:
We are still missing translations for other languages. Usually we just use google translate, we had little errors with it. Currently we support `en-US`, `pt-BR`, `es-ES` and `cat-CAT`

here is the Portuguese one: `Escolha ao menos {{count}} items`